API_CHANGES.md

```diff
@@ -10,11 +10,11 @@ Add in the linux `task.get_threads` method added to the API.
 
 2.0.3
 =====
-`DEVICE_OBJECT.get_attached_devices` and `DRIVER_OBJECT.get_devices` added to the API.
+Add in the windows `DEVICE_OBJECT.get_attached_devices` and `DRIVER_OBJECT.get_devices` method added to the API.
 
 2.0.2
 =====
-Fix the behaviour of the offsets returned by the PDB scanner.
+Fix the behavior of the offsets returned by the PDB scanner.
```

Comment:
This is an American spelling, rather than a typo.  I'm not intending to ensure all documentation is consistent between Amertican/British english, so I'll include this, but if it were the only change I'd reject it.

Reply:
All right, this is often the case with global projects, I think I was not familiar with British English. This case is not wrong, so I will revert it.